added: github-actions docs (#130)
mia-platform/documentation: docs/development_suite/deploy/pipeline-configuration.md

sidebar_label: Configure a Project Pipeline
---
Projects created with the Mia-Platform Console can be deployed with simple script and tooling for

> applying standard Kubernetes yaml file that describe the various resources. For aiding the developer we have created

Reviewer: ```suggestion
applying standard Kubernetes yaml files that describe the various resources. To aid the developer we have created
```

- **STAGE_TO_DEPLOY**: set to the current environment of the project to deploy
- **RELEASE_DATE**: current date generated with the unix command `date -I'seconds' -u`  

> Additionally, to these variables the console will generate a `.env` file for every environment of the project that can

Reviewer: ```suggestion
Additionally, to these variables, the Console will generate a `.env` file for every environment of the project that can
```
